src/server.h

```diff
@@ -2363,6 +2364,7 @@ struct pendingCommand {
     size_t argv_len_sum;      /* Sum of lengths of objects in argv list. */
     unsigned long long input_bytes;
     struct redisCommand *cmd;
+    getKeysResult keys_result;
```

Comment:
i see you added it, but don't see it being set or used

Reply:
Yes, I'm still trying to add it and not introducing the break change of the command filter.